flocker/node/_docker.py

```diff
@@ -508,6 +530,12 @@ def _remove():
                     break
 
             try:
+                # The ``docker.Client.stop`` method puts the container into
+                # a stopping state, but the container cannot be removed
+                # until the container has actually stopped running.
+                while self._blocking_container_runs(container_name):
```

Comment:
Put a comment like the one 10ish lines below saying that this is not covered by tests.